Podfile

```diff
@@ -10,7 +10,11 @@ target :MockUtils do
 end
 
 def shared_pods
-    pod 'MongoSwift', '= 0.0.9'
+    pod 'MongoSwift', '= 0.1.0'
+end
+
+def mongo_mobile
+    pod 'MongoMobile', :git => 'https://github.com/mongodb/swift-mongo-mobile/', :branch => 'master'
```

Comment:
[minor] we should not point to master since this could break at any time without warning